app/server/tests/api-getCommunitiesByTitle.test.js

```diff
@@ -165,4 +114,39 @@ describe('Get Community by Title - With Login, testing with new Community after
   test('API returns description -  "Invalid token."',() => {
     expect(response.text).toBe("Community not found.");
   });
+});
+
+
+
+describe('Creating new Community with " " as title', () => {
```

Comment:
This doesn't seem right to me. Should the app be allowing a community with an empty string as a title?

Reply:
Right now it does. I added that test because if you create a community with " " as the title, and then find that community by title, it responds with all communities in the database.

Reply:
I will need to check for this